Author: Julien Cristau <jcristau@debian.org>
Description: Change the disallowedWindowOps default
 Upstream introduced a new 'disallowedWindowOps' resource to fine-tune
 which operations are affected by the 'allowWindowOps' setting.  After
 discussion with the security team, change the default for the new
 resource to include all operations.

---
 xterm.man |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/xterm.man
+++ b/xterm.man
@@ -3263,8 +3263,8 @@ This is a comma-separated list of names,
 from \fIdtterm\fP the operation number).
 The default value is
 .NS
-20,21,SetXprop,SetSelection
-(i.e., all except a few \*(``dangerous\*('' operations are allowed).
+1,2,3,4,5,6,7,8,9,11,13,14,18,19,20,21,GetSelection,SetSelection,SetWinLines,SetXprop
+(i.e., no operations are allowed).
 .NE
 .IP
 The names are listed below.
